overlord/snapstate/snapmgr.go

```diff
@@ -457,9 +435,86 @@ func (m *SnapManager) ensureRefreshes() error {
 	}
 	chg.Set("snap-names", updated)
 	chg.Set("api-data", map[string]interface{}{"snap-names": updated})
+
+	m.state.EnsureBefore(0)
```

Comment:
now that this is called from Ensure directly without the timer indirection I think we don't need this anymore